.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,43 @@
+name: Benchmark Main
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
```

Comment:
Instead of having two workflow files (one for backfilling and one for updates to main), can we have a single workflow which takes an optional tag as an argument, defaulting to use the main branch if not set?

Reply:
I can, but I worry that would give the impression that it would work on any tag when it can only work with 1.6+. Is this important to you?  It seems that would be a lot more work involved in backfilling all the versions I enumerated here. We don't even need to keep this long term. It can be deleted after a successful run.

Reply:
Not important and we can iterate. If it turns out that we want to backfill performance runs on a recurring basis, let's consider refactoring some sort reusuable work flow (i.e. [reusable-markdown-link-check.yml](https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/reusable-markdown-link-check.yml)).